src/workerd/api/queue.h

```diff
@@ -32,30 +32,48 @@ class WorkerQueue: public jsg::Object {
     // contentType determines the serialization format of the message.
     jsg::Optional<kj::String> contentType;
 
-    JSG_STRUCT(contentType);
+    // The number of seconds to delay the delivery of the message being sent.
+    jsg::Optional<uint> delaySecs;
+
+    JSG_STRUCT(contentType, delaySecs);
     JSG_STRUCT_TS_OVERRIDE(QueueSendOptions {
       contentType?: QueueContentType;
+      delaySecs ?: number;
```

Comment:
And why the the extra space between `delaySecs` and the question mark?

Reply:
I even tried `clang-format` with the style file (.clang-format) used in `edgeworker`, and it always adds me that space there. I removed it anyway to match the original style.

Reply:
Looks like the issue is that these `JSG_*` blocks are explicitly excluded from clang-format: https://github.com/cloudflare/workerd/blob/main/.clang-format

So it's just not changing anything regardless of what you put here.